src/guide/component-custom-events.md

```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
 
 ## Handling `v-model` modifiers
 
-In 2.x, we have hard-coded support for modifiers like `.trim` on component `v-model`. However, it would be more useful if the component can support custom modifiers. In 3.x, modifiers added to a component `v-model` will be provided to the component via the modelModifiers prop:
+In Vue 2.x, we had hard-coded support for modifiers like `.trim` or `.native` on component `v-model` bindings. And even though Vue 3.x also supports [out of the box modifiers](/guide/forms.html#modifiers), it would be more useful if our components could support custom modifiers of our own. 
```

Comment:
**suggestion**: I think we should remove 2.x mention here completely and just mention we have built-in modifiers with the link as nothing changes except `.native` modifier.

Also, let's remove the statement about custom modifiers being **more** useful (it really depends on the use-case). What about just `useful`?

```suggestion
As we already mentioned, `v-model` has [built-in modifiers](/guide/forms.html#modifiers) like `.trim` or `.native`. In some cases, you might also want to add your own custom modifier.
```